### PR TITLE
docs(API): expose `ElementTemplates#get`

### PR DESCRIPTION
**Pull request by nikku**
Tell people about the fact that `ElementTemplates#get` is a thing.

### CODE REVIEW

File: docs/API.md [hunk 1]
```diff
@@ -4,26 +4,24 @@ The core element template functionality is exposed through the [`ElementTemplate
 
 > If you [create your own element templates provider](https://github.com/bpmn-io/element-templates/blob/main/docs/CREATE_ELEMENT_TEMPLATE_PROVIDER.md) then you must at least implement the `ElementTemplates` interface and exposed it as the named `elementTemplates` service.
 
-### `ElementTemplatesLoader#setTemplates(templates)`
+### `ElementTemplates#get(diagramElement): Template|null`
 
-Validates and sets element templates synchronously. Does not throw on template errors but instead emits them through an `elementTemplates.errors` event.
+Get the element template currently applied to a diagram element. Returns `null` if no template is applied or the applied template is not known to the modeler.
```
Comment (smbea) [inline docs/API.md:9]:
```suggestion
Get the element template currently applied to a diagram element. Returns `null` if no template is applied or if the applied template is not known to the modeler.
```

File: docs/API.md [hunk 2]
```diff
@@ -4,26 +4,24 @@ The core element template functionality is exposed through the [`ElementTemplate
...
+const template = elementTemplates.get('My_Task');
+```
 
-const loader = modeler.get('elementTemplatesLoader');
+### `ElementTemplates#get(templateId, templateVersion?): Template|null`
 
-const templates = [
-  { id: 'some template', properties: [ ... ] }
-];
+Get the element template with the given id and optional version. If no version is provided it defaults to latest. Returns `null` if no template is found matching the search criteria.
```
Comment (smbea) [inline docs/API.md:19]:
```suggestion
Get the element template with the given id and optional version. If no version is provided it defaults to latest. Returns `null` if no template matches the search criteria.
```

**Review by smbea — COMMENTED** [review]
I suggested two small changes that make sense to me. But if it's just me, feel free to ignore

**Comment by nikku** [comment]
Thanks for the review. I incorporated your suggestions.

**Pull request merged**